kernel/src/table_configuration.rs

```diff
@@ -400,27 +400,12 @@ impl TableConfiguration {
     /// [`TableChanges`]: crate::table_changes::TableChanges
     #[internal_api]
     pub(crate) fn is_cdf_read_supported(&self) -> bool {
-        static CDF_SUPPORTED_READER_FEATURES: LazyLock<Vec<TableFeature>> =
-            LazyLock::new(|| vec![TableFeature::DeletionVectors]);
-        let protocol_supported = match self.protocol.reader_features() {
-            // if min_reader_version = 3 and all reader features are subset of supported => OK
-            Some(reader_features) if self.protocol.min_reader_version() == 3 => {
-                ensure_supported_features(reader_features, &CDF_SUPPORTED_READER_FEATURES).is_ok()
-            }
-            // if min_reader_version = 1 and there are no reader features => OK
-            None => self.protocol.min_reader_version() == 1,
-            // any other protocol is not supported
-            _ => false,
-        };
+        let protocol_supported = self.protocol.is_cdf_supported();
         let cdf_enabled = self
             .table_properties
```

Comment:
Hmm I'm realizing we never disallow ICT here, but we don't currently read it. I looked through the protocol doesn't say anything about cdf...


> readers should use the inCommitTimestamp as the commit timestamp for operations like time travel and [DESCRIBE HISTORY](https://docs.delta.io/latest/delta-utility.html#retrieve-delta-table-history).

Reply:
yea (discussed a little offline) let's take a follow-up (no worse than today and many of the ICT requirements in spec are _should_s)